bin/installers/Linux/installer.sh

```diff
@@ -6,13 +6,13 @@ SCRIPT_DIR=$(dirname "${BASH_SOURCE[0]}")
 INSTALL_DIR=$(cd -P -- "$SCRIPT_DIR/.." && pwd -P)
 source "$INSTALL_DIR/../../lib/utils"
 
-setup_configuration "$OCI_DIR" "$HOME/strong-oracle.rc"
+setup_configuration "$OCI_DIR" "$HOME/oracledb.rc"
 cat << MYEOF
 -------------------------------------------------------------------------------
-"strong-oracle.rc" has been created in \$HOME. Please manually add the
+"oracledb.rc" has been created in \$HOME. Please manually add the
```

Comment:
~/.bash_profile is only run for login shells, isn't it? I always start my shells as login shells, but last time I saw, that wasn't true of new terminals with iTerm or most multi-tab terminal emulators. Or perhaps I misremember.